my-app/src/Navbar/Badges.jsx

```diff
@@ -8,105 +11,250 @@ export default function Badges({ watchedVideosCount }) {
+      setBlackholegotten(false);
     }
     if (watchedVideosCount >= 15) {
       setSunbagdegotten(true);
+    } else {
+      setSunbagdegotten(false);
     }
     if (watchedVideosCount >= 20) {
       setEarthbadgegotten(true);
+    } else {
+      setEarthbadgegotten(false);
     }
     if (watchedVideosCount >= 25) {
       setMoonbagdegotten(true);
+    } else {
+      setMoonbagdegotten(false);
     }
     if (watchedVideosCount >= 30) {
       setMeteoritebadgesgotten(true);
+    } else {
+      setMeteoritebadgesgotten(false);
     }
   });
 
   return (
-    <div>
-      <h1>Badges </h1>
+    <div className="theWhole">
+      <h1 className="headerWords">
+        Badges <LiaIdBadgeSolid />
+      </h1>
+
       <div className="generalDiv">
-        <div className="achievementLabel">
+        <div className="badges">
           <img
             alt="challengePatches"
             className="badgeCategoryImage"
             src="https://cdn.kastatic.org/images/badges/master-challenge-blue-medium.png"
+            style={{ width: "200px", height: "200px" }}
           />
-          {challengePatchesgotten && (
-            <h1>Congratulations on achieving the Challenge Patch ! </h1>
-          )}
-          Challenge Patches
+          <h4 className="words">Challenge Patches</h4>
+
+          <button className="dropDownBtn" onClick={handleBadges}>
+            Click here to see your progress
+            <IoMdArrowDropdownCircle size={30} color="white" />
+          </button>
+
+          <div>
+            {challengePatchesgotten ? (
+              <p>
+                Congrats! You've earned the Challenge Patches badge. Challenge
+                Patches are special awards for completing the first 5 video
+                challenges.
+              </p>
+            ) : (
+              <p>{showmessage ? "You are not there yet. Keep going!" : ""}</p>
+            )}
+          </div>
         </div>
 
-        <div className="achievementLabel">
+        <div className="badges">
           <img
             alt="blackHoleBadges"
             className="badgeCategoryImage"
             src="https://cdn.kastatic.org/images/badges/eclipse-medium.png"
+            style={{ width: "200px", height: "200px" }}
           />
-          {blackholegotten && (
-            <h1>Congratulations on achieving the Black Hole Badge ! </h1>
-          )}
-          Black Hole Badges
+
+          <h4 className="words">Black Hole Badges</h4>
+          <button className="dropDownBtn" onClick={handleBadges}>
+            Click here to see your progress{" "}
+            <IoMdArrowDropdownCircle size={30} color="white" />
+          </button>
+
+          <div>
+            {blackholegotten ? (
+              <p>Congrats! You've earned the Black Hole Badges. </p>
+            ) : (
+              <p>
+                {showmessage
+                  ? "Black Hole badges are a symbol of deep knowledge and mastery. Keep exploring!"
+                  : ""}
```

Comment:
Another way of writing this out could be:
`{showmessage && "Black Hole badges are a symbol of deep knowledge and mastery. Keep exploring!"}`

This essentially is doing something nearly identical to what you have, but instead of displaying an empty message (an empty string) it will instead display nothing at all. Basically the text would only render if the `showMessage` variable is false. In contrast, what you have always renders a text, but it the `showMessage` variable is false then the text is an empty string.

No need to change all of the usages here, but this is something to play with in the future!